README.md

```diff
@@ -123,6 +123,14 @@ interactive: True
 # Runs the command in reporting mode. The standard output of the remote command will
 # be dumped to the screen instead of the exit code.
 report: True
+
+# The following list form the arguments to the command line. They are set as bash shell
```

Comment:
I think this line could do with some clearing up - either 'The following list form**s** the arguments **sent** to the command line' or 'The following list form**s** the arguments **for** the command line'

Reply:
https://github.com/alces-software/adminware/pull/175/commits/e7a7726910586fd962ec5d2b07b1c9e550b1e4c8